bait-files/src/main/webapp/success.jsp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Form Successfully Submitted</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../success.css">
+  </head>
+  <body>
+    <img src="https://img.icons8.com/cotton/128/000000/checkmark.png" alt="Green checkmark with circle around it"></img>
+    <h1>Thank You!</h1>
+    <p>Your submission has been received and our team will review it soon. Please check back for the completed review with your results.</p>
+    <br></br>
+    <a id="home" alt="Black and white home icon" href="https://step-bait-project-2020.uc.r.appspot.com/history.jsp"><img src="https://img.icons8.com/ios/50/000000/home-page.png"></img></a>
+    <p>Click above to return to the home page</p>
```

Comment:
Maybe name this to something else Since we don’t exactly have a home page? Something like  “... your request portal.” maybe?

Reply:
i like that!